website/docs/overview/overview.md

```diff
@@ -49,50 +49,51 @@ for using them.
-| [Pbzip2](https://microsoft.github.io/VirtualClient/docs/workloads/compression/pbzip2.md) | Compression | linux-x64, linux-arm64 | [BSD](http://compression.great-site.net/pbzip2/)  |
-| [PostgreSQL](https://microsoft.github.io/VirtualClient/docs/workloads/postgresql//postgresql.md) | Relational Database Performance | linux-x64, linux-arm64, win-x64 | [PostgreSQL](https://www.postgresql.org/about/licence/)
-| [Prime95](https://microsoft.github.io/VirtualClient/docs/workloads/prime95/prime95.md) | CPU Stress | linux-x64 | [Custom](https://www.mersenne.org/legal/)  |
-| [Redis](https://microsoft.github.io/VirtualClient/docs/workloads/redis/redis.md) | In-Memory Data Cache | linux-x64, linux-arm64 | [BSD-3 (Redis)](https://github.com/redis/redis/blob/unstable/COPYING)<br/>[GPL-2.0 (Memtier)](https://github.com/RedisLabs/memtier_benchmark/blob/master/COPYING)  |
-| [SockPerf](https://microsoft.github.io/VirtualClient/docs/workloads/network-suite/network-suite.md) | Network Latencies | linux-x64, linux-arm64 | [Custom](https://github.com/Mellanox/sockperf/blob/sockperf_v2/copying)  |
-| [SPEC CPU 2017, SPECrate Integer](https://microsoft.github.io/VirtualClient/docs/workloads/speccpu/) | Precision CPU, Integer Calculations | linux-x64, linux-arm64 | [End User License Required](https://www.spec.org/cpu2017/Docs/licenses.html) |
-| [SPEC CPU 2017, SPECrate Floating Point](https://microsoft.github.io/VirtualClient/docs/workloads/speccpu/) | Precision CPU, Floating-point Calculations | linux-x64, linux-arm64 | [End User License Required](https://www.spec.org/cpu2017/Docs/licenses.html) |
-| [SPEC CPU 2017, SPECspeed Integer](https://microsoft.github.io/VirtualClient/docs/workloads/speccpu/) | Precision CPU, Integer Calculations | linux-x64, linux-arm64 | [End User License Required](https://www.spec.org/cpu2017/Docs/licenses.html) |
-| [SPEC CPU 2017, SPECspeed Floating Point](https://microsoft.github.io/VirtualClient/docs/workloads/speccpu/) | Precision CPU, Floating-point Calculations | linux-x64, linux-arm64 | [End User License Required](https://www.spec.org/cpu2017/Docs/licenses.html) |
-| [SPEC JBB 2015, SPECjbb](https://microsoft.github.io/VirtualClient/docs/workloads/specjbb/) | Java Server | linux-x64, linux-arm64, win-x64, win-arm64 | [End User License Required](https://www.spec.org/jbb2015/) |
-| [SPEC JVM 2008, SPECjvm](https://microsoft.github.io/VirtualClient/docs/workloads/specjvm/specjvm.md) | Java Runtime Performance | linux-x64, linux-arm64, win-x64, win-arm64 | [SPEC](https://www.spec.org/spec/docs/SPEC_General_License.pdf)  |
-| [SPEC Power 2008, SPECpower](https://microsoft.github.io/VirtualClient/docs/workloads/specpower/) | High precision, steady-state CPU usage | linux-x64, linux-arm64, win-x64, win-arm64 | [End User License Required](https://www.spec.org/power_ssj2008/) |
+| [7zip](../workloads/compression/7zip.md) | Compression | linux-x64, linux-arm64 | [GNU LGPL](https://www.7-zip.org/faq.html) |
```

Comment:
Could you revert the absolute path references change and just add your hadoop documents?